tests/functional/run-action-big-data/setUp.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types = 1);
+
+use MongoExtractor\FunctionalTests\DatadirTest;
+use MongoExtractor\Tests\Traits\ImportDatasetTrait;
+use Symfony\Component\Filesystem\Filesystem;
+
+return static function (DatadirTest $test): void {
+    $fs = new Filesystem;
+    $datasetFilepath = sprintf('%s/../../datasets/big-dataset.json', __DIR__);
+    $expectedCsvFilepath = sprintf('%s/expected/data/out/tables/big-data.csv', __DIR__);
+    $fs->remove($datasetFilepath);
+    $fs->remove($expectedCsvFilepath);
+
+    $fs->dumpFile($datasetFilepath, "[\n");
+    $fs->dumpFile($expectedCsvFilepath, "\"id\",\"name\",\"email\",\"phone\"\r\n");
+
+    $numOfRows = 100000; // 100K
```

Comment:
zkoušel si to třeba i s 1M/10M?

Reply:
jj, jen to pak žuchne na paměti v tom testu, při diffu těch CSV

Reply:
ale extractor proběhne?

Reply:
jj

Reply:
tak paráda 👍